GliaWidgets/SecureConversations/Confirmation/SecureConversations.ConfirmationView.swift

```diff
@@ -159,9 +169,9 @@ extension SecureConversations {
         }
 
         private func renderProps() {
-            header.title = props.style.headerTitle
-            header.backButton.tap = props.backButtonTap.execute
-            header.closeButton.tap = props.closeButtonTap.execute
+            header.props.title = props.style.headerTitle
```

Comment:
I would suggest to avoid mutating props directly, since this is implicit change for data.

Reply:
fixed - 997017ba6c73b2161cee6aef5164eb16e074d9d6